Update branded_linux_desktop_clients.adoc (#52)
owncloud/docs-client-branding: modules/ROOT/pages/branded_desktop_client/branded_linux_desktop_clients.adoc

# You can customitze the main html file and re-run repo-admin.py later.
# For a server without https support use --insecure --url http://...
#
set -euxo pipefail

cd "$(dirname "${0}")"

opt=()

# if a .repo-admin-prefix file is found in a parent directory, --url can be omitted

> opt+=("--url" "https://minio.owncloud.services/clients/build-linux/7270/repo/")

Author: ```suggestion
opt+=("--url" "http://download.example.com/repo")
```

python3 bin/repo-admin.py  "${opt[@]}" -d 'download' -p '.*-client' -i 'index.html' -f ..
----

> The url after unpacking is an internal minio url, or a non working example like `\http://download.example.com/repo`.

Reviewer: We try not to include any internally available URLs anymore. It also doesn't really matter whether there's a broken URL in place or none at all.

> The url after unpacking is an internal minio url, or a non working example like `\http://download.example.com/repo`.

Reviewer 2: ```suggestion
The url after unpacking is an invalid url.
```

No actual example url in the text needed. Users can see it in the `example.sh`.

> You must change example.sh before it can be run.

Reviewer: Should be copied, I guess. It's an `example.sh` after all.

Author: What does it mean "should be copied"? 
My intention was to
a) do not expose an internal URL
b) make it more obvious that this is an example. (rather than a magic constant)

Best solution: have this line only as a comment, so that no editing is needed at all, when using the .prefix file.

You must change example.sh before it can be run.

* Either
   - replace the url with the correct url leading to the exact folder in which the archive was unpacked +
     (ending in `.../mycloud-2.10.0.6752-linux/` here).
* Or 
   - Delete (or comment out) the line containing the url.
   - Make sure there is a `.repo-admin-prefix` file in the parent folder of the unpacked archive (here next to `mycloud-2.10.0.6752-linux`)
   - Its contents is the base URL leading up to this parent folder. +
     E.g. if the correct --url parameter were `http://cloud.example.com/install/mycloud-2.10.0.6752-linux/` +

>      then the prefix file can be created like this:

Reviewer 3: > then the prefix file can be created like this:

I am missing the part coming after `this`...
Or did I miss something?

Author: "... created like this" refers to the next two items that follow below.
